File: Facebook.NET/GraphAPI.PaymentPricePoint.cs
using System;
using System.Runtime.Serialization;

namespace Facebook
{
    public static partial class GraphAPI
    {
        [DataContract]
        public sealed class PaymentPricePoint
        {
            [DataMember(Name = "mobile")]
            public PricePoint[] Mobile { get; set; }
        }
    }
}

File: Facebook.NET - VB/Contracts/GraphAPI.PaymentPricePoint.vb
Imports System
Imports System.Runtime.Serialization

Namespace Facebook
    Public NotInheritable Partial Class GraphApi
        <DataContract()>
        Public NotInheritable Class PaymentPricePoint
            <DataMember(Name:="mobile")>
            Public Property Mobile() As GraphApi.PricePoint()
        End Class
    End Class
End Namespace
